Code/GraphMol/MolDraw2D/DrawTextFTQt.cpp

```diff
@@ -0,0 +1,107 @@
+//
+//  Copyright (C) 2020 Greg Landrum and T5 Informatics GmbH
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+//
+#include <QPainter>
+#include <QPainterPath>
+
+#include <GraphMol/MolDraw2D/DrawTextFTQt.h>
+
+namespace RDKit {
+
+// ****************************************************************************
+DrawTextFTQt::DrawTextFTQt(double max_fnt_sz, double min_fnt_sz,
+                           const std::string &font_file, QPainter &qp)
+    : DrawTextFT(max_fnt_sz, min_fnt_sz, font_file), d_qp(qp) {}
+
+// ****************************************************************************
+double DrawTextFTQt::extractOutline() {
+  dp_qpp.reset(new QPainterPath());
+  double adv = DrawTextFT::extractOutline();
+  const auto &col = colour();
+  QColor this_col(int(255.0 * col.r), int(255.0 * col.g), int(255.0 * col.b),
+                  int(255.0 * col.a));
+
+  QPen pen(this_col);
+  pen.setJoinStyle(Qt::RoundJoin);
+  pen.setColor(this_col);
+  d_qp.setPen(pen);
```

Comment:
It is used elsewhere, but the rest of the code should already be handling that particular problem since the drawing code runs across multiple rendering platforms, not all of which support equivalents to QPainter's save() and restore() methods